webpack.common.js

```diff
@@ -0,0 +1,73 @@
+require("dotenv").config();
+
+const { resolve } = require("path");
+
+const ESLintPlugin = require("eslint-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+// const CopyWebpackPlugin = require("copy-webpack-plugin");
```

Comment:
Consider removing commented code from the final version of your code before creating PRs.
That also includes you comments for various code blocks in Russian - it's better to use English in case you'll ever want to share it with someone.